.github/workflows/eints-to-git.yml

```diff
@@ -0,0 +1,58 @@
+name: Eints to Git
+
+# This script has to be in master, despite the code being executed in the
```

Comment:
There is also https://github.com/OpenTTD/workflows
Is that an option?

Reply:
It is. But I was thinking it would be wiser to have this as close to the eints repo as possible, as it really is related to eints. But this whole repo with its branches is a bit of an odd duck ofc :) I checked if we can fork a repo inside the same organization, but you cannot. Which is a bit sad, tbh :P Do you prefer it in a completely different repository?